File: activities/ParImpar/parImpar.c
#include <stdio.h>
#include <stdlib.h>

int parImpar(int a) {
    if (a % 2 == 0)
        return 0;
}

int main() {

    int a,b;

    printf("Digite o número para verificar se é par ou impar: ");
    scanf ("%d", &a);

    if (parImpar(a) == 0) 
        printf("O número: %d é par \n",a);
    else
        printf("O número: %d é impar \n",a);

}

File: activities/ParImpar/ParImpar.java
import java.util.Scanner;

public class ParImpar {
    
    public static int parImpar(int a) {
        if (a % 2 == 0)
            return 0;
        else
            return 1;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int a;

        System.out.print("Digite o número para verificar se é par ou impar: ");
        a = scanner.nextInt();

        if (parImpar(a) == 0)
            System.out.println("O número " + a + " é Par");
        else
            System.out.println("O número " + a + " é Ímpar");


        scanner.close();
    }
}
